docs/source/manual/core.rst

```diff
@@ -1257,89 +1257,6 @@ This gets converted into this JSON:
         "first_name": "Coda"
     }
 
-.. _man-core-representations-validation:
-
-Validation
-----------
-
-You can add validation annotations on resource endpoints similar to how it is done in
-:ref:`man-core-configuration`. These annotations can be placed on resource return values,
-``*Param`` annotations, and fields of your representation classes.
-
-For example, if we're accepting client-provided ``Person`` and returning a modification, we probably
-want to ensure that the ``name`` field of the object isn't ``null`` or blank on both the request and
-response. We can do this as follows:
-
-.. code-block:: java
-
-    public class Person {
-
-        @NotEmpty // ensure that name isn't null or blank
-        private final String name;
-
-        @JsonCreator
-        public Person(@JsonProperty("name") String name) {
-            this.name = name;
-        }
-
-        @JsonProperty("name")
-        public String getName() {
-            return name;
-        }
-    }
-
-Then, in our resource class, we can add the ``@Valid`` annotation to the ``Person`` parameter so
-that we know the ``Person`` object has a name in our endpoint. In addition, adding the ``@Valid``
-annotation to the return value will ensure we're not returning invalid data back to clients:
-
-.. code-block:: java
-
-    @PUT
-    @Valid
-    public Person replace(@Valid Person person) {
-        // Oops, we got a bug!
-        return new Person(null);
-    }
-
-    @GET
-    @Valid
-    public Person find(@QueryParam("name") @NotEmpty name) {
-        // ...
-    }
-
-If validation fails, Dropwizard will return a ``text/plain`` response with an HTTP status code
-depending on where the validation failed:
-
-* Violation on a resource return value causes a ``500 Internal Server Error``
-* Violation on a representation class causes a ``422 Unprocessable Entity``
-* Violation on a ``*Param`` annotation causes a ``400 Bad Request``
-
-So in our example, if the ``name`` field is missing in a ``PUT`` request, Dropwizard will return a
-``422 Unprocessable Entity`` response detailing the validation error::
-
-    * name may not be empty
-
-.. _man-core-resources-validation-advanced:
-
-Advanced
-********
-
-More complex validations (for example, cross-field comparisons) are often hard to do using
-declarative annotations. As an emergency maneuver, add the ``@ValidationMethod`` to any
-``boolean``-returning method which begins with ``is``:
-
-.. code-block:: java
-
-    @ValidationMethod(message="may not be Coda")
```

Comment:
I would add the @JsonIgnore property to the ValidationMethod. See this pull request: https://github.com/dropwizard/dropwizard/pull/1108

Without the tag, erroneous properties are added to the json during testing (when following the testing documentation).